torba/torba/server/db.py

```diff
@@ -543,17 +543,15 @@ def read_utxo_state(self):
                 raise self.DBError('failed reading state from DB')
             self.db_version = state['db_version']
             if self.db_version not in self.DB_VERSIONS:
-                raise self.DBError('your UTXO DB version is {} but this '
-                                   'software only handles versions {}'
-                                   .format(self.db_version, self.DB_VERSIONS))
+                raise self.DBError(f'your UTXO DB version is {self.db_version} but this '
+                                   f'software only handles versions {self.DB_VERSIONS}')
             # backwards compat
             genesis_hash = state['genesis']
             if isinstance(genesis_hash, bytes):
                 genesis_hash = genesis_hash.decode()
             if genesis_hash != self.coin.GENESIS_HASH:
-                raise self.DBError('DB genesis hash {} does not match coin {}'
-                                   .format(genesis_hash,
-                                           self.coin.GENESIS_HASH))
+                raise self.DBError(f'DB genesis hash {genesis_hash} does not '
+                                   f'match coin {self.coin.GENESIS_HASH}')
```

Comment:
Does it fit on single line?

Reply:
@eukreign Similar to the case of torba/torba/server/db.py . Awaiting your guidance.

Reply:
https://github.com/lbryio/lbry-sdk/blob/9bb0ca2f0d7ef428d17ee8bdec89d0bb17f99dca/torba/setup.cfg#L16

Reply:
@eukreign Line length would become 114. So, doesn't fit in one line.